companies/joinclubhouse.json

```diff
@@ -10,14 +10,16 @@
     "runs": [
         "Clubhouse"
     ],
-    "address": "5335 Lawton Avenue\nOakland\nCalifornia 94618\nUnited States of America",
+    "address": "Alpha Exploration Co.\n548 Market St\nPMB 72878\nSan Francisco, California 94104-5401",
```

Comment:
Can't find any sources for the new address. I've emailed them to confirm the correct address.

Reply:
@mal-tee Never got an answer from them, but California Secretary of State documents were filed with the change of address, we're good to merge.

Can we add the doc as a source too? 
https://businesssearch.sos.ca.gov/Document/RetrievePDF?Id=04563531-29903880

Reply:
> Can we replace the current link with this one, as well?
> [businesssearch.sos.ca.gov/Document/RetrievePDF?Id=04563531-29903880](https://businesssearch.sos.ca.gov/Document/RetrievePDF?Id=04563531-29903880)

That's an interesting point that we currently don't have guidelines on. For now, I'll replace the old source but I have opened #1042 to discuss this for the future.

Reply:
I already did that :P